pkg/event/target/webhook.go

```diff
@@ -94,16 +95,27 @@ func (target *WebhookTarget) HasQueueStore() bool {
 
 // IsActive - Return true if target is up and active
 func (target *WebhookTarget) IsActive() (bool, error) {
-	u, pErr := xnet.ParseHTTPURL(target.args.Endpoint.String())
-	if pErr != nil {
-		return false, pErr
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	req, err := http.NewRequest(http.MethodHead, target.args.Endpoint.String(), nil)
+	if err != nil {
+		if xnet.IsNetworkOrHostDown(err) {
```

Comment:
I don't think http.NewRequest() can return a network or host down error here

Reply:
@vadmeste the error can be `url.Error` which is why we used IsNetworkOrHostDown() to indicate as not connected. because we have no other top-level handling for other errors.